lavalink/AudioTrack.py

```diff
@@ -16,6 +16,7 @@ def build(self, track, requester):
             self.title = track['info']['title']
             self.uri = track['info']['uri']
             self.requester = requester
+            self.preferences = kwargs

```

Comment:
What's `preferences` for?

Reply:
Preferences per track, this could be also used in the master.
Example use:
  - There is a track for which we don't want to send a "now playing" message, when the event is called we'd check the preferences if we can send the message. Like `_user_data` in DefaultPlayer.

Reply:
Sounds good